### ISSUE TITLE
How to ignore file leaks with deno test

### ISSUE THREAD

**Issue by abdurahmanshiine** [denoland/deno#27454]
Hey guys,

I keep getting this error 
```bash
error: Leaks detected:
  - A file was opened during the test, but not closed during the test. Close the file handle by calling `file.close()`.
```
The issue is it's not telling me which file it is, or which file object it's referring to, so I was wondering how can I disable leaks tracing in testing, I honestly don't even need that at this point.

Version: Deno 2.1.4

**Comment by kt3k** [comment]
You can disable resource leak detection by passing `sanitizeResources: false` option to `Deno.test`

https://docs.deno.com/runtime/fundamentals/testing/#resource-sanitizer

```ts
Deno.test("my test", { sanitizeResources: false }, async () => {
  const file = await Deno.open("README.md");
  // now it's fine to leave the file open
});
```

**Comment by abdurahmanshiine** [comment]
Hey @kt3k,
Thanks for your response. I'm using the @std/testing package so I can write the tests in jest style. Can I pass the same option to it?

**Comment by kt3k** [comment]
Yes, you can pass `sanitizeResources: false` to `describe` or `it`:

```js
import { describe, it } from "jsr:@std/testing/bdd";

describe("foo", { sanitizeResources: false }, () => {
  it("bar", async () => {
    const file = await Deno.open("README.md");
  });
});
```

Alternatively, there's unstable API for disabling leak detection for the entire file:

```js
import { describe, it } from "jsr:@std/testing/bdd";
import { configureGlobalSanitizers } from "jsr:@std/testing/unstable-bdd";

configureGlobalSanitizers({
  sanitizeResources: false,
});

describe("foo", () => {
  it("bar", async () => {
    const file = await Deno.open("README.md");
  });
});
```

This might look cleaner if there's many test cases in a single file.

FYI there's also a similar option `sanitizeOps: false` which disables the detection of async operation leaks such as `setTimeout`. You might be also interested in disabling this.

**Comment by dandv** [comment]
> The issue is it's not telling me which file it is

@abdurahmanshiine Does [`--trace-leaks`](https://github.com/denoland/deno/issues/27948) help you discover where the leak came from?

**Comment by maxmckenzie** [comment]
@abdurahmanshiine I'm not sure disabling the leak protection is a good idea. Even with the test files

the docs for Deno's resource sanitizer states to use file.close() after

https://docs.deno.com/runtime/fundamentals/testing/#resource-sanitizer

```
const file = await Deno.open("hello.txt");
// Do something with the file
file.close(); // <- Always close the file when you are done with it
```

Hope that helps.
